debian/po/pt_BR.po

```diff
@@ -465,12 +463,11 @@ msgid "The following package(s) will be REMOVED:"
 msgstr ""
 
 #: ../../uaclient/messages/__init__.py:148
-#, fuzzy
 msgid "The following package(s) will be reinstalled from the archive:"
-msgstr "Você não tem pacotes instalados de terceitos."
+msgstr "O(s) pacote(s) à seguir serão reinstalados:"
 
```

Comment:
This translation is incorrect. Before removing a `fuzzy` translation, check that it is correct.